e2e-tests/vm/runner-cloud-cfg.yaml

```diff
@@ -0,0 +1,66 @@
+#cloud-config
+password: E2E_VM_PASSWORD
+ssh_pwauth: true
+users:
+  - name: ubuntu
+    sudo: ['ALL=(ALL) NOPASSWD:ALL']
+    lock_passwd: false
+    gecos: Ubuntu
+    groups: sudo
+    homedir: /home/ubuntu
+    shell: /bin/bash
+chpasswd:
+  expire: false
+  users:
+    - {name: ubuntu, password: E2E_VM_PASSWORD, type: text}
+
+package_update: true
+package_upgrade: true
+packages:
+  - build-essential
+  - openssh-server
+  - ubuntu-desktop-minimal
+  - systemd-container
+
+write_files:
+  - path: /etc/dconf/profile/user
+    owner: root:root
+    permissions: '0644'
+    content: |
+      user-db:user
+      system-db:local
+
+  # Disable screen lock and sleep
+  - path: /etc/dconf/db/local.d/00-e2e-tests-requirements
+    owner: root:root
+    permissions: '0644'
+    content: |
+      [org/gnome/settings-daemon/plugins/power]
+      sleep-inactive-ac-type='nothing'
+      sleep-inactive-ac-timeout=0
+
+      [org/gnome/desktop/session]
+      idle-delay=uint32 0
+
+runcmd:
+  # Disable apport
+  - sed -i 's/enabled=1/enabled=0/' /etc/default/apport
+  - systemctl mask --now apport.service
+  # Disable automatic updates and remove some unnecessary packages
+  - apt-get remove -y update-manager gnome-initial-setup
+  # We don't need Firefox for the tests, so we can remove it to save some space
+  - snap remove firefox --purge
+  # Configure SSH to allow interactive authentication
+  - sed -i 's/KbdInteractiveAuthentication no/KbdInteractiveAuthentication yes/' /etc/ssh/sshd_config
```

Comment:
Maybe can use the exact code that is in https://github.com/ubuntu/authd/blob/main/docs/reference/cloud-init-deploy.md ?